### PR TITLE
Adapt sidebar to TopicStore and Forums

### PR DESCRIPTION
**Pull request by mjlescano**

### CODE REVIEW

**Comment by slifszyc** [comment]
1) [This line](https://github.com/DemocracyOS/app/blob/fix-sidebar/lib/homepage/homepage.js#L30) is no longer valid, as not a regular staff should see the 'create first topic' button, only the forum owner. It's the staff member when `multiForum === false`

2) The button link is not correct, it's linking to `/admin/topics/create` instead of `/my-forum/admin/topics/create`

**Comment by slifszyc** [comment]
3) After updating a topic, when entering again to edit it, the previous version of it is shown (but it is updated on the admin list) `multiForum === true`
4) Similar thing happens when un/publishing it. It gets updated on that page, but doesn't show/hide on the forum page

**Comment by slifszyc** [comment]
3b) Same thing happens on `multiForum === false`

File: lib/topic-api/index.js [hunk 1]
```diff
@@ -445,12 +432,12 @@ app.post('/topic/:id/publish', restrict, hasAccess, function (req, res) {
...
               log('Error when sending notification for event %s', eventName);
             } else {
-              log('Successfully notified publishing of topic %s', topicDoc.id);
+              log('Successfully notified publishing of topic %s', topic.id);
             }
           });
       }
 
-      res.json(expose(topicKeys)(topicDoc.toJSON()));
+      res.json(exposeTopic(topic, req.user));
```
Comment (gvilarino) [inline lib/topic-api/index.js]: Hey @slifszyc, @mjlescano is it right to write a response before the topic gets saved? It didn't use to be like this (bejore there used to be `res.json(200)` inside the callback for the `topic.save` call.

Comment (mjlescano) [inline lib/topic-api/index.js]: It is inside the `topic.save(function (err, saved) {` callback, see https://github.com/DemocracyOS/app/pull/985#discussion-diff-36030130R411

Comment (gvilarino) [inline lib/topic-api/index.js]: Then I guess it's wrong on what I saw when rebasing [my PR](https://github.com/DemocracyOS/app/pull/995/files#diff-193a8ef887bba2089d3469292e6d666bR434). I guess I should've looked closer. Please be sure to check my PR as well

**Pull request merged**